.buildkite/pipeline.yml

```diff
@@ -20,6 +20,7 @@ steps:
     branches: master
     agents:
       queue: native
```

Comment:
I know they are the build node tags but you only need `docker: builder` and we can get rid of the `native` queue.